admin/partials/page-bulk-export.php

```diff
@@ -2,25 +2,28 @@
 /**
  * Publish to Apple News partials: Bulk Export page template
  *
- * phpcs:disable VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable
- *
- * @global array $articles
- *
  * @package Apple_News
  */
 
+// Expect these variables to be defined in Admin_Apple_Bulk_Export_Page::build_page() but make sure they're set.
+$apple_page_title       ??= __( 'Bulk Export', 'apple-news' );
+$apple_page_description ??= __( 'The following articles will be affected.', 'apple-news' );
+$apple_posts            ??= [];
+$apple_submit_text      ??= __( 'Go', 'apple-news' );
```

Comment:
the old hard-coded text tells me that "Bulk Export" will publish the posts to Apple News. This new text does not tell me that. If I saw "Bulk Export", I would expect it to download a file vs publishing. Maybe "Bulk Export" needs to be "Bulk Publish"?

Reply:
The current language was retained and is supposed to be applied here: https://github.com/alleyinteractive/apple-news/blob/83b8bece248684d34167d92b7c58ce8ee403f7ee/admin/class-admin-apple-bulk-export-page.php#L86-L115

With the changes in this PR, this partial should only ever be rendered when a known bulk action is being invoked, and the text should reflect that action (which happens in the lines linked above). 

There isn't any circumstance where the default strings seen here should actually be rendered, but I included them so that there would be something on the page if it ever happened. You're right that the strings are vague, but how the page would behave at that point is also not known, so I wasn't sure what else to do.

Reply:
cool, I was kind of picking that up from the code. this makes sense. I still wonder about using `Export` vs `Upload` or `Publish`, but maybe that's a question for project maintainers or stakeholders. The button text was `Publish All` and I don't see that it is retained, but maybe I'm missing it. Anyway, I'm good with whatever, so feel free to proceed as you see fit.

Reply:
Thanks for catching that the button text wasn't actually kept! Whoops. Fixed in 0ab4d9c8551b19a510fd9ce54bec8d0d358981b1.

Re. the word `Export`, I agree that it isn't as clear as it could be. I'm reluctant to change it here, but, as it happens, @scottnelle has been working on refining that language in #1204 (see also https://l.alley.dev/a3a948a289), so maybe we could expand that PR to include language for this page that will be consistent with the change being made on the list table.